test/e2e/specs/published-content/reader__view.ts

```diff
@@ -18,28 +17,25 @@ declare const browser: Browser;
 describe( DataHelper.createSuiteTitle( 'Reader: View' ), function () {
 	let page: Page;
 	let readerPage: ReaderPage;
-	const user: TestAccountName =
+	const accountName: TestAccountName =
 		envVariables.JETPACK_TARGET === 'remote-site' ? 'jetpackRemoteSiteUser' : 'commentingUser';
-	const expectedSiteName =
-		envVariables.JETPACK_TARGET === 'remote-site'
-			? SecretsManager.secrets.testAccounts.jetpackRemoteSiteUser.testSites?.primary.url
-			: SecretsManager.secrets.otherTestSites.notifications;
 	let testAccount: TestAccount;
 
 	beforeAll( async function () {
 		page = await browser.newPage();
-		testAccount = new TestAccount( user );
+		testAccount = new TestAccount( accountName );
 		await testAccount.authenticate( page );
 	} );
 
-	it( 'View the Reader stream', async function () {
+	it( 'Visit the Reader', async function () {
 		readerPage = new ReaderPage( page );
 		await readerPage.visit();
-		const siteOfLatestPost = await readerPage.siteOfLatestPost();
-		expect( siteOfLatestPost ).toEqual( expectedSiteName );
 	} );
 
-	it( 'Visit latest post', async function () {
-		await readerPage.visitPost( { index: 1 } );
+	it( 'Reader stream is present', async function () {
+		await Promise.any( [
+			page.getByRole( 'link', { name: 'Find sites to follow' } ),
+			page.getByRole( 'article' ),
+		] );
```

Comment:
I updated the expectations and opted to use "raw" selectors in interest of time.